Libraries/Components/TabBarIOS/TabBarIOS.ios.js

```diff
@@ -20,19 +20,22 @@ const ViewPropTypes = require('ViewPropTypes');
 
 var requireNativeComponent = require('requireNativeComponent');
 
-class TabBarIOS extends React.Component<{
-  style?: $FlowFixMe,
-  unselectedTintColor?: $FlowFixMe,
-  tintColor?: $FlowFixMe,
-  unselectedItemTintColor?: $FlowFixMe,
-  barTintColor?: $FlowFixMe,
+import type {StyleObj} from 'StyleSheetTypes';
+import type {ViewProps} from 'ViewPropTypes';
+
+class TabBarIOS extends React.Component<ViewProps & {
+  style?: StyleObj,
+  unselectedTintColor?: string,
+  tintColor?: string,
+  unselectedItemTintColor?: string,
+  barTintColor?: string,
   barStyle?: 'default' | 'black',
   translucent?: boolean,
   itemPositioning?: 'fill' | 'center' | 'auto',
+  children: React.Element<any> | React.Element<any>[],
```

Comment:
This should probably be [`React.Node`](https://flow.org/en/docs/react/children/)